exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/grpc/ManagedChannelUtil.java

```diff
@@ -42,34 +44,47 @@
    *
    * @throws SSLException if error occur processing the certificates
    */
-  public static void setTrustedCertificatesPem(
-      ManagedChannelBuilder<?> managedChannelBuilder, byte[] trustedCertificatesPem)
+  public static void setClientKeysAndTrustedCertificatesPem(
+      ManagedChannelBuilder<?> managedChannelBuilder, @Nullable byte[] privateKeyPem,
+      @Nullable byte[] privateKeyChainPem, byte[] trustedCertificatesPem)
       throws SSLException {
     requireNonNull(managedChannelBuilder, "managedChannelBuilder");
     requireNonNull(trustedCertificatesPem, "trustedCertificatesPem");
 
     X509TrustManager tmf = TlsUtil.trustManager(trustedCertificatesPem);
+    X509KeyManager kmf = null;
+    if (privateKeyPem != null && privateKeyChainPem!=null) {
```

Comment:
Looks like there's a formatting issue, i.e. it should be `privatekKeyChainPem != null`.

You can run `./gradlew spotlessApply` to apply the formatting conventions.

Reply:
done